mount-helper/Makefile

```diff
@@ -48,6 +52,30 @@ deb-build:
 	cp *.deb ./install && rm -f *.deb
 	rm -rf $(BUILD_DIR)
 
+iks-deb-build:
+	rm -rf $(IKS_BUILD_DIR)
+
+	python3 $(PYTHON_MERGE_SCRIPT) ./src $(IKS_BUILD_DIR)/$(MOUNT_SCRIPT)
+	chmod 755 $(IKS_BUILD_DIR)/$(MOUNT_SCRIPT)
+
+	mkdir -p $(IKS_BUILD_DIR)/DEBIAN
+	cp install/postinst.sh $(IKS_BUILD_DIR)/DEBIAN/postinst
+	cp install/prerm.sh $(IKS_BUILD_DIR)/DEBIAN/prerm
+	cp -r install/certs $(IKS_BUILD_DIR)/tmp
+	cp install/share.conf $(IKS_BUILD_DIR)/
+	cp install/charon-log.conf $(IKS_BUILD_DIR)/tmp
+
+	echo "Package: $(IKS_BUILD_DIR)" > $(IKS_CONTROL)
+	echo "Version: $(IKS_APP_VERSION)" >> $(IKS_CONTROL)
+	echo "Maintainer: $(MAINTAINER)" >> $(IKS_CONTROL)
+	echo "Architecture: $(DEB_ARCH)" >> $(IKS_CONTROL)
+	echo "Description: $(DESCRIPTION)" >> $(IKS_CONTROL)
+	echo "Depends: python3(>= $(MIN_PYTHON_VER)), strongswan-swanctl(>= $(MIN_STRONGSWAN_VER)), charon-systemd, nfs-common" >> $(IKS_CONTROL)
+
+	dpkg-deb --build $(IKS_BUILD_DIR)
+
+	rm -rf $(IKS_BUILD_DIR)
+
```

Comment:
We are not building IKS packages within mount helper container right? So I don't think we need this as it will be covered in Makefile under mount-helper-container